pkg/codegen/templates/fiber/fiber-register.tmpl

```diff
@@ -5,15 +5,18 @@ type FiberServerOptions struct {
 }
 
 // RegisterHandlers creates http.Handler with routing matching OpenAPI spec.
-func RegisterHandlers(router fiber.Router, si ServerInterface) {
+func RegisterHandlers(router *fiber.App, si ServerInterface) {
   RegisterHandlersWithOptions(router, si, FiberServerOptions{})
 }
 
 // RegisterHandlersWithOptions creates http.Handler with additional options
-func RegisterHandlersWithOptions(router fiber.Router, si ServerInterface, options FiberServerOptions) {
+func RegisterHandlersWithOptions(router *fiber.App, si ServerInterface, options FiberServerOptions) {
```

Comment:
Is this needed? fiber.Router should have `.Use` as well

Reply:
Doh! You're right 👍